logic/src/commonMain/kotlin/com/wire/kalium/logic/data/featureConfig/FeatureConfigMapper.kt

```diff
@@ -0,0 +1,8 @@
+package com.wire.kalium.logic.data.featureConfig
+
+import com.wire.kalium.network.api.featureConfigs.FeatureConfigResponse
+
+class FeatureConfigMapper {
+    fun fromFileSharingResponse(featureConfigResponse: FeatureConfigResponse): FileSharingModel =
```

Comment:
what do you think of renaming it to `fromFileSharingDTO`? we have the same convention as other mappers.